sunsynk/sensor.py

```diff
@@ -197,6 +249,10 @@ def update_value(self) -> None:
         sval = str(self.reg_value[0])
         self.value = f"{sval[:-2]}:{sval[-2:]}"
 
+    def value_to_reg(self, value: Any) -> int | Tuple[int, ...]:
+        """Get the reg value from a display value."""
+        raise NotImplementedError()
```

Comment:
This has already been implemented in another branch as part of some larger `TimeRWSensor` enhancements. This exception will not be hit because `TimeRWSensor` is not writable for now in run.py.